projects/m1/040-password-validation/js/index.js

```diff
@@ -0,0 +1,23 @@
+const prompt = require ("prompt-sync")();
+
+//8 char, lowercase, uppercase, number
+const pass = prompt("Insert a valid password: ");
+
+function passValid(){
+    if (pass.length < 8){
+        return false;
+    }
+    if (!/[A-Z]/.test(pass)){
+        return false;
+    }
+    if (!/[a-z]/.test(pass)){
+        return false;
+    }
+    if (!/[\d]/.test(pass)){
+        return false;
+    }
+    return true
+}
+
+
+console.log(passValid());
```

Comment:
If I wanted to test the function (by **importing it** elsewhere), what would be **the call I should make**?